pageserver/src/tenant/timeline/span.rs

```diff
@@ -0,0 +1,26 @@
+#[cfg(not(debug_assertions))]
+#[inline]
+pub(crate) fn debug_assert_current_span_has_tenant_and_timeline_id() {}
+
+#[cfg(debug_assertions)]
+#[inline]
```

Comment:
Could get rid of these `#[inline]` at the same time, it does nothing for `pub(crate)` just noise in the source code.

Reply:
Not on this PR.